README.md

```diff
@@ -108,6 +108,22 @@ cargo build --release
 
 Note that compilation is a memory intensive process. We recommend having 4 GiB of physical RAM or swap available (keep in mind that if a build hits swap it tends to be very slow).
 
+#### Build from Source with Docker
+
+You can also build from source using 
+[Parity CI docker image](https://github.com/paritytech/scripts/tree/master/dockerfiles/ci-linux):
+
+```bash
+git checkout <latest tagged release>
+docker run --rm -it -w /shellhere/polkadot \
```

Comment:
What is `shellhere` supposed to be?

Reply:
It's just a name of a working directory inside the container without any additional meaning.